app/src/main/kotlin/edu/stanford/bdh/engagehf/messages/MessageAction.kt

```diff
@@ -0,0 +1,12 @@
+package edu.stanford.bdh.engagehf.messages
+
+sealed interface MessageAction {
+    data object UnknownAction : MessageAction
+    data class VideoAction(val video: Video) : MessageAction
+    data object MedicationsAction : MessageAction
+    data object MeasurementsAction : MessageAction
+    data class QuestionnaireAction(val questionnaireId: String) : MessageAction
+    data object HealthSummaryAction : MessageAction
+}
+
+data class Video(val sectionId: String, val videoId: String)
```

Comment:
If this is used in context of message action only, I would be more in favor of simply declaring the properties in video action type instead of introducing this new type

Reply:
sounds good to me honestly